src/WebApp/Pages/Questions.cshtml

```diff
@@ -0,0 +1,66 @@
+﻿@page "/{category}/questions"
+@model Codidact.Core.WebApp.Pages.Posts.QuestionsModel
+@using Codidact.Core.Application.Questions.Queries
+
+@{
+    ViewData["Title"] = "Questions";
+    var query = Model.Query;
+}
+
+<partial name="_ValidationSummary" />
+
+
+<div class="grid has-margin-4">
+    <div class="grid--cell is-flexible">
+        <h1 class="has-margin-0">@Model.Result.Category.DisplayName Questions</h1>
+        <h3>@Model.Result.Category.ShortExplanation</h3>
+    </div>
+    <div class="grid--cell">
+        <a asp-page="questions/new" class="button is-medium has-margin-4 is-filled">New question</a>
+    </div>
+</div>
+<div class="button-list is-gutterless has-margin-top-4 has-float-right">
+    <a asp-page="questions" asp-route-sort="@QuestionsQuerySortType.Best" class="button is-muted is-outlined @(query.Sort == QuestionsQuerySortType.Best?"is-active":"")">best</a>
```

Comment:
You're missing some spacing in the ternary expression in the `class` attribute here (and likewise in the next two lines).

Reply:
Could've been something like:
```
    @foreach (var sortType in (QuestionsQuerySortType[]) Enum.GetValues(typeof(QuestionsQuerySortType)))
    {
        <a asp-page="questions" asp-route-sort="@sortType" class="button is-muted is-outlined @(query.Sort == sortType ? "is-active" : "")">@sortType</a>
    }
```

But for now just formatted the condition